fbw-a32nx/src/systems/instruments/src/MCDU/mcdu.scss

```diff
@@ -176,7 +176,7 @@ span .s-text {
 
 .b-text {
   font-family: HoneywellMCDU !important;
-  letter-spacing: normal;
+  letter-spacing: -6px;
```

Comment:
Won't this break the layout since it only applies to large text?

Reply:
Probably not. If you were trying to get the gap down the sides of the text to match photos, this is unfortunately the wrong fix. The problem is the aspect ratio of the display in the 3D model.